sql/src/main/java/io/crate/operation/reference/doc/lucene/FetchIdCollectorExpression.java

```diff
@@ -55,14 +58,23 @@ public void setNextReader(LeafReaderContext context) {
         docBase = context.docBase;
     }
 
+    @VisibleForTesting
+    static FetchIdCollectorExpression build(long readerId, int docBase, byte relationId) {
+        FetchIdCollectorExpression instance = new FetchIdCollectorExpression();
+        instance.readerId = readerId;
+        instance.docBase = docBase;
+        instance.relationId = relationId;
+        return instance;
+    }
+
     /**
-     * Pack jobSearchContextId and doc integers into 1 long.
+     * Pack readerId, doc and relationId into 1 long.
```

Comment:
pls describe how it is packed in words instead/or additionally to the reverse logic